code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -731,3 +732,89 @@ datum/reagent/medicine/insulin/on_mob_life(var/mob/living/M as mob)
 	M.reagents.remove_reagent("sugar", 3)
 	..()
 	return
+
+/*
+	Alright, so, let's get one thing straight here.
+
+	The first 4 reagents never ever show up outside of medibots, mediborgs, and sleepers.
+
+	EVER. No recipes, no secret monkey techniquing it, nothing.
+
+	If they do, I'm gonna nerf them.
+
+	The 5th reagent never shows up outside of mediborgs and that fucking speshul snoflaek Mysterious Medibot bullshit.
+	If it does, it's going bye bye and you get to use styptic patch appliers or some shit.
+	Whatever I end up coding to replace it, idfk. Either way, you know the rules now. Happy healing!
+
+	- Iamgoofball
+*/
+
+datum/reagent/medicine/brute_solution
+	name = "Brute Solution"
+	id = "brute_solution"
+	description = "You shouldn't ever see this."
```

Comment:
Not true, you can produce it by injecting monkeys/corpses and grinding the meat. Obviously you can't then synth it, but it can concievably make its way into the chemmaster